datahub/company/serializers.py

```diff
@@ -207,6 +209,51 @@ class Meta:
             f'company.{ContactPermission.view_contact_document}': 'archived_documents_url_path',
         }
 
+    def _notify_consent_service(self, validated_data):
+        """
+        Trigger the update_contact_consent task with the current version
+        of `validated_data`. The actual enqueuing of the task happens in the
+        on_commit hook so it won't actually notify the consent service unless
+        the database transaction was successful.
+        """
+        if 'accepts_dit_email_marketing' not in validated_data:
+            # If no consent value in POST body
```

Comment:
I think this comment could be clearer as it could be a POST or PATCH. Maybe 'request body' would be more generic.
```suggestion
            # If no consent value in request body
```